Extensions/DirectXMathAVX.h

```diff
@@ -28,7 +28,7 @@ inline bool XMVerifyAVXSupport()
 
     // See http://msdn.microsoft.com/en-us/library/hskdteyh.aspx
     int CPUInfo[4] = {-1};
-#ifdef __clang__
+#if defined(__clang__) || defined(__GNUC__)
```

Comment:
From what I understand clang/LLVM also defines ``__GNUC__`` so these could all be simplified to:

```
#ifdef __GNUC__
```

Correct?

Reply:
I'm not sure about Clang/C2 compiler, looks like it should not have such definition. (By the way, is it supported by DirectXMath at all?)

Reply:
Clang/C2 was experimental and is now abanonded, and it never implemented any of the SSE intrinsics. It 'kinda' worked with ``_XM_NO_INTRINSICS_`` but Clang/C2 was mostly about improving conformance of source.

Reply:
Well, looks like Clang on Windows with MSVC flavour (`clang-cl.exe`) (at least, which is builtin with MSVS 2019 16.3) does not defines `__GNUC__`. 

So we should use both definitions for CPUID.